### PR TITLE
DDF-3699 Upgraded Openlayers to 4.6.5

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Upgrades Openlayers to 4.6.5 
https://github.com/openlayers/openlayers/blob/ca3e11a10b47ba0323b3fbc477ce1d39eb06b26e/changelog/v4.6.0.md
Should help with migrating to OL 5.0 in the future.
I did a search and replace where the old version (3.20.1) and "openlayers" was being used. Could use feedback on if other places need to be upgraded/tested. I didn't change the webjar since that seems to be related to the Standard Search UI and this is focused on Intrigue. I'm not sure if keeping the webjar will still cause an NSP failure.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@djblue @mojogitoverhere  @ricklarsen 
#### Select relevant component teams: 
@codice/ui 
#### Choose 2 committers to review/merge the PR. 
@codice/ui 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@pklinef

#### How should this be tested? (List steps with links to updated documentation)
I did a build and manually tested (ingest, query, move around inspector/map) Intrigue, but could use feedback on specific tests
#### What are the relevant tickets?
[DDF-3699](https://codice.atlassian.net/browse/DDF-3699)
#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/3092)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -236,7 +236,7 @@
         <jbake.maven.plugin.version>0.0.9</jbake.maven.plugin.version>
         <jruby.version>1.7.26</jruby.version>
 
-        <openlayers.version>3.20.1</openlayers.version>
+        <openlayers.version>4.6.5</openlayers.version>
```
Comment (shaundmorris) [inline pom.xml]: This change will update the map-standards documentation

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
I verified NSP passes

**Comment by shaundmorris** [comment]
build now

**Comment by andrewkfiedler** [comment]
I put the rest of the changes we need in here: https://github.com/codice/ddf/compare/master...andrewkfiedler:pr-3092?expand=1

**Comment by shaundmorris** [comment]
Sweet, thanks @andrewkfiedler! I'll incorporate those in this PR.

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
@andrewkfiedler can you think any more specific testing we should do on this PR?

**Comment by shaundmorris** [comment]
Also verified the map loads on Windows

**Pull request merged**